### DISCUSSION TITLE
Shuffle or rando?

### DISCUSSION THREAD

**Discussion by secretmeister2** [TigerX6/Ultimate-Randomizer#1] [General]
Does this mod shuffle loot tables between blocks, or does it just choose a random item to drop?
I am trying to find a loot shuffler for 1.21

**Comment by TigerX6** [comment]
The plugin will determine a random item to be dropped when a block is broken for the first time. This item is saved for each block so that it stays the same every time you break that block until you use `/shuffle`. It works similarly with mobs, however, here the amount of different items dropped and the drop chance stay the same, but with random items. 
Many things can also be configured in the config, such as if mobs/blocks should drop random loot, the amount of items to be dropped, etc.
